docs/source/getting_started/installation.md

```diff
@@ -25,23 +25,23 @@ The Elyra [JupyterLab extensions](https://jupyterlab.readthedocs.io/en/stable/us
 * [Python 3.8+](https://www.python.org/downloads/) (or later)
 
 JupyterLab dependencies:
+ - Elyra >= 4.0.0 requires JupyterLab 4.x
  - Elyra >= 2.0.0 requires JupyterLab 3.x
 
 The instructions below are installing the latest release.
 
 ### Packaging
 
-Prior to version 3.1, the `elyra` package included all dependencies. Subsequent releases allow for selective dependency installation:
+You can install elyra with all optional dependencies or with specific dependencies:
```

Comment:
I think this here changed recently, see and compare with 
[make kfp dependency optional PR that made it into main](https://github.com/elyra-ai/elyra/pull/3248)
and specifically this documentation for readthedocs:

https://github.com/elyra-ai/elyra/pull/3248/files#diff-7b84ea4a0a5058af570a2d621f94cf210464b0e8ce04c6d7bd24c614b38e0abb